app/src/main/res/drawable/splash.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:drawable="@color/colorPrimary"/>
+
+    <item>
+        <bitmap
+            android:gravity="center"
+            android:src="@mipmap/ic_launcher"/>
```

Comment:
Looking into the current issue and here is the offending line.  One way I know of fixing it is by putting an ic_launcher image with the drawable folder, and reference that with @drawable/ic_launcher.  I was trying to look into the mipmap, but it seems like there are not a lot of resources on it.